plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/transformers/EpochTimeAttributeTransformer.java

```diff
@@ -94,20 +97,123 @@ DateTimeFormatter getFormatter() {
             }
 
             @Override
-            long toRawValue(Instant instant) {
-                return instant.getEpochSecond() * 1_000_000_000 + instant.getNano();
+            Long toRawValue(Instant instant) {
+                return instant.getEpochSecond() * GIGA + instant.getNano();
+            }
+        },
+
+        // https://docs.microsoft.com/en-us/dotnet/api/system.datetime.ticks?view=net-6.0#remarks
+        dotnet {
+            @Override
+            Instant toInstant(Number value) {
+                return ticksToInstant(value.longValue(), DOTNET_TICKS_OFFSET);
+            }
+
+            @Override
+            DateTimeFormatter getFormatter() {
+                return DOTNET_TICKS_FORMATTER;
+            }
+
+            @Override
+            Long toRawValue(Instant instant) {
+                return instantToTicks(instant, DOTNET_TICKS_OFFSET);
+            }
+        },
+
+        // https://docs.microsoft.com/en-us/dotnet/api/system.datetime.fromfiletimeutc?view=net-6.0#system-datetime-fromfiletimeutc(system-int64)
+        w32filetime {
+            @Override
+            Instant toInstant(Number value) {
+                return ticksToInstant(value.longValue(), W32_FILETIME_OFFSET);
+            }
+
+            @Override
+            DateTimeFormatter getFormatter() {
+                return DOTNET_TICKS_FORMATTER;
+            }
+
+            @Override
+            Long toRawValue(Instant instant) {
+                return instantToTicks(instant, W32_FILETIME_OFFSET);
+            }
+        },
+
+        // https://docs.microsoft.com/en-us/dotnet/api/system.datetime.fromoadate?view=net-6.0#system-datetime-fromoadate(system-double)
+        oadate {
+            @Override
+            Instant toInstant(Number value) {
+                return daysToInstant(value.doubleValue(), OADATE_OFFSET);
+            }
+
+            @Override
+            DateTimeFormatter getFormatter() {
+                return NANOS_FORMATTER;
+            }
+
+            @Override
+            Double toRawValue(Instant instant) {
+                return instantToDays(instant, OADATE_OFFSET);
+            }
+        },
+
+        // https://www.sqlite.org/lang_datefunc.html
+        sqliteJulian {
+            @Override
+            Instant toInstant(Number value) {
+                return daysToInstant(value.doubleValue(), SQLITE_JULIAN_OFFSET);
+            }
+
+            @Override
+            DateTimeFormatter getFormatter() {
+                return NANOS_FORMATTER;
```

Comment:
From what I see, SQLite's REAL data, when processed with `strftime(..., 'julianday')`, have only ~10 us precision perhaps due to the large offset. I'll check math and pick a more concise format